src/Avalonia.Styling/Controls/PseudoClassesExtensions.cs

```diff
@@ -8,7 +8,7 @@ public static class PseudolassesExtensions
         /// Adds or removes a pseudoclass depending on a boolean value.
         /// </summary>
         /// <param name="classes">The pseudoclasses collection.</param>
-        /// <param name="name">The name of the pseudoclass to set.</param>
+        /// <param name="name">The name of the pseudoclass to set. Name need to follow this rules: != null, starts with a ':' and length > 2 (including the ':' char)</param>
```

Comment:
> length > 2

This comment is incorrect I think? The check on line 228 is `name.Length < 2" which when inverted would be `name.Length >= 2`.

To be honest though I'm not sure this addition to the comments is necessary: we'll be adding nullable reference types to this assembly soon, and the "`:` followed by something" rule should be obvious.

Reply:
ok i will remove the comment from here as you said (and you're right)
>": followed by something" rule should be obvious